src/services/aggregationService.ts

```diff
@@ -0,0 +1,28 @@
+
+import { v4 as uuid } from 'uuid';
+import { DATABASE_NAME, Tables } from '../db/db.constants';
+import { executeQuery } from '../db/queryExecuter';
+import { GraphConfig } from '../types/graph.types';
+import { excelService } from './excelService';
+import _ from 'lodash';
+
+class AggregationService {
+    tableName = Tables.dashboards;
+
+    async runAggregation(graphConfig: GraphConfig, dataSourceId: string) {
+        const table = await excelService.getDataBySourceId(dataSourceId);
+        const series: { name: string; y: number; }[] = [];
+        if (graphConfig.y_field.aggragation === "sum") {
```

Comment:
this needs to be changed in the future, but for now I don't care

Reply:
yes yes